domain/src/test/java/org/oppia/android/domain/classify/rules/dragAndDropSortInput/DragDropSortInputIsEqualToOrderingWithOneItemAtIncorrectPositionClassifierProviderTest.kt

```diff
@@ -26,27 +24,54 @@ import kotlin.test.fail
-  private val ITEM_SET_1_AB = listOf("item a", "item b")
-  private val ITEM_SET_1_ABC = listOf("item a", "item b", "item c")
-  private val ITEM_SET_1_A = listOf("item a")
-  private val ITEM_SET_2_ITEM_2 = listOf("item 2")
-  private val ITEM_SET_3_ITEM_3 = listOf("item 3")
-  private val SET_LIST_ITEMS_AB_ITEM_2_ITEM_3 =
-    createListOfSetsOfHtmlStrings(ITEM_SET_1_AB, ITEM_SET_2_ITEM_2, ITEM_SET_3_ITEM_3)
-  private val SET_LIST_ITEMS_ABC_ITEM_2_ITEM_3 =
-    createListOfSetsOfHtmlStrings(ITEM_SET_1_ABC, ITEM_SET_2_ITEM_2, ITEM_SET_3_ITEM_3)
-  private val SET_LIST_ITEM_2_ITEM_3_ITEMS_AB =
-    createListOfSetsOfHtmlStrings(ITEM_SET_2_ITEM_2, ITEM_SET_3_ITEM_3, ITEM_SET_1_AB)
-  private val SET_LIST_ITEM_A_ITEM_2_ITEM_3 =
-    createListOfSetsOfHtmlStrings(ITEM_SET_1_A, ITEM_SET_2_ITEM_2, ITEM_SET_3_ITEM_3)
+
+  private val NON_NEGATIVE_VALUE_TEST_0 =
+    InteractionObjectTestBuilder.createNonNegativeInt(value = 0)
+
+  private val SET_ITEM_A =
+    InteractionObjectTestBuilder.createListOf("item a")
+
+  private val SET_ITEM_A_B =
+    InteractionObjectTestBuilder.createListOf("item a", "item b")
+
+  private val SET_ITEM_A_B_C =
+    InteractionObjectTestBuilder.createListOf("item a", "item b", "item c")
+
+  private val SET_ITEM_2 =
+    InteractionObjectTestBuilder.createListOf("item 2")
+
+  private val SET_ITEM_3 =
+    InteractionObjectTestBuilder.createListOf("item 3")
+
+  private val LIST_OF_SETS_AB_2_3 =
+    InteractionObjectTestBuilder.createListOfSetsOfHtmlStrings(
+      InteractionObjectTestBuilder.createHtmlStringList(SET_ITEM_A_B, SET_ITEM_2, SET_ITEM_3)
+    )
+
+  private val LIST_OF_SETS_ABC_2_3 =
+    InteractionObjectTestBuilder.createListOfSetsOfHtmlStrings(
+      InteractionObjectTestBuilder.createHtmlStringList(SET_ITEM_A_B_C, SET_ITEM_2, SET_ITEM_3)
+    )
+
+  private val LIST_OF_SETS_2_3_AB =
+    InteractionObjectTestBuilder.createListOfSetsOfHtmlStrings(
+      InteractionObjectTestBuilder.createHtmlStringList(
+        InteractionObjectTestBuilder.createHtmlStringList(SET_ITEM_2, SET_ITEM_3, SET_ITEM_A_B)
+      )
+    )
+
+  private val LIST_OF_SETS_A_2_3 =
+    InteractionObjectTestBuilder.createListOfSetsOfHtmlStrings(
+      InteractionObjectTestBuilder.createHtmlStringList(SET_ITEM_A, SET_ITEM_2, SET_ITEM_3)
+    )
 
   @Inject
   internal lateinit var dragDropSortInputIsEqualToOrderingWithOneItemAtIncorrectPositionClassifierProvider: // ktlint-disable max-line-length
     DragDropSortInputIsEqualToOrderingWithOneItemAtIncorrectPositionClassifierProvider
 
   private val isEqualToOrderingWithOneItemIncorrectClassifier: RuleClassifier by lazy {
-    dragDropSortInputIsEqualToOrderingWithOneItemAtIncorrectPositionClassifierProvider.createRuleClassifier() // ktlint-disable max-line-length
+    dragDropSortInputIsEqualToOrderingWithOneItemAtIncorrectPositionClassifierProvider
+      .createRuleClassifier() // ktlint-disable max-line-length
```

Comment:
no need of ktlint disable here, as this line is not having char more than 100 chars.

Reply:
try to setup ktlint at your local as per mentioned in our wiki, this will prevent ktlint failure in PR. 
https://github.com/oppia/oppia-android/wiki/Ktlint-Guide